music21/abcFormat/testFiles.py

```diff
@@ -831,7 +831,8 @@ def testAbc21Chords(self):
         for abc_chord in ['[]', '[z]']:
             ah = af.readstr(abc_dl + '[]')
             s = translate.abcToStreamScore(ah)
-            self.assertFalse(s.elements[1].getElementsByClass(chord.Chord),
+            part = s.getElementsByClass(stream.Part)
```

Comment:
this is returning a Stream iterator -- you probably want a specific part, like:

```
part = s.getElementsByClass(stream.Part)[0]
```

which is abbreviated as:

```
part = s.parts[0]
```

Reply:
Thanks,
I fixed this and  notice an  additional Bug, i guess made by the translater. (I still have to research this)

for abc_chord in ['[]', '[z]']:
    ah = af.readstr(abc_dl + '[]')
    s = translate.abcToStreamScore(ah)
    chord = list(s.parts[0].notes)
    print(abc_chord, chord, ah.tokens)

[] [<music21.note.Note C>] [<music21.abcFormat.ABCMetadata 'L:1/8'>, <music21.abcFormat.ABCChord '[]'>]
[z] [<music21.note.Note C>] [<music21.abcFormat.ABCMetadata 'L:1/8'>, <music21.abcFormat.ABCChord '[]'>]

There is no Chord at all but a note C in the part. The C is not present in abc code nor token list at all.